src/js/database.js

```diff
@@ -450,9 +452,9 @@ function DataManager(db, resourceDir, sourceDir) {
             }
         },
 
-        getAllWords: function (dict) {
+        getAllWords: function (language_id, dict) {
             var mythis = this;
-            var container = "en_" + dict + "_tw";
+            var container = language_id + "_" + dict + "_tw";
```

Comment:
Might be a good place to use interpolation.

Reply:
Fixed.